### ISSUE TITLE
Can we access parsed OpenApi schema after validation is done?

### ISSUE THREAD

**Issue by lezhnev74** [lezhnev74/openapi-psr7-validator#23]
Validation logic parses and caches `\cebe\openapi\spec\OpenApi ` internally. It uses it for validation purposes, but this instance (which is already in memory) can be valuable for later use AFTER validation is done.

I have this use case in my head:

1. We built a validator from the yaml file.
2. The schema is cached somewhere and used inside.
3. After validation is complete, can we somehow continue use this finder to maybe find extra details about the matched request or response?
3.1 RequestValidator returns OperationAddress, maybe we can use it to find extra details in the schema?

In other words, can we access the parsed `\cebe\openapi\spec\OpenApi` from the validator after the validation is done?

**Comment by lezhnev74** [comment]
We could make this method public: `\OpenAPIValidation\PSR7\ValidatorBuilder::getOrCreateSchema`. @scaytrase

**Comment by scaytrase** [comment]
I hope we should reuse schema of validator itself, not of the builder

чт, 23 мая 2019 г., 18:18 Dmitry Lezhnev <notifications@github.com>:

**Comment by lezhnev74** [comment]
Maybe we can add this interface to validators:
```php
interface ReusableSchema {
    public function getSchema(): \cebe\openapi\spec\OpenApi;
}
```

**Comment by scaytrase** [comment]
sounds good

чт, 23 мая 2019 г., 18:33 Dmitry Lezhnev <notifications@github.com>:
